app/models/pet.rb

```diff
@@ -95,6 +95,9 @@ class Pet < ApplicationRecord
     query.where(published: true).order(:breed).distinct.pluck(:breed)
   }
 
+  # Returns an array of unique species of an organization
+  scope :unique_species, -> { distinct.pluck(:species) }
```

Comment:
Do you think this should also limit the species where the pet is adoptable? For example, if there are cats and dogs available for adoption, then all cats are adopted, do we still want to shot the view cats button? Or should we just show buttons for pets where there are currently adoptable animals?

Reply:
yeah, I think to only show adoptable animals.